components/typography/style/index.ts

```diff
@@ -54,6 +54,17 @@ const genTypographyStyle: GenerateStyle<TypographyToken> = (token) => {
         userSelect: 'none',
       },
 
+      [`&${componentCls}-style-code`]: {
```

Comment:
```suggestion
      [`&${componentCls} > code`]: {
```

What about that?

Reply:
The class (`ant-typography-style-code`) is for `<span />`. not `<code />` 👀

<img width="532" alt="image" src="https://user-images.githubusercontent.com/28584349/222485216-db615bc1-d012-4a9e-b2a1-7d4a34581d05.png">